make/Makefile.dev

```diff
@@ -0,0 +1,88 @@
+DOCKER_REPO?=quay.io/openshiftio
```

Comment:
I think we should get rid out of `openshiftio`

Reply:
Yes, but we do not have any better repo  yet. Will switch to something more suitable later.